docs/spec/staking/state.md

```diff
@@ -90,14 +90,14 @@ type Validator struct {
     ProposerRewardPool sdk.Coins    // reward pool collected from being the proposer
     
     // TODO: maybe this belongs in distribution module ?
-	LastBondedTokens   sdk.Rat     // last bonded token amount
+	LastBondedTokens   sdk.Dec     // last bonded token amount
 }
 
 type CommissionInfo struct {
-    Rate        sdk.Rat  // the commission rate of fees charged to any delegators
-    Max         sdk.Rat  // maximum commission rate which this validator can ever charge
-    ChangeRate  sdk.Rat  // maximum daily increase of the validator commission
-    ChangeToday sdk.Rat  // commission rate change today, reset each day (UTC time)
+    Dece        sdk.Dec  // the commission rate of fees charged to any delegators
```

Comment:
Probably want a better variable name here

Reply:
was a find replace error - changed back to `Rate`